daml-script/test/src/com/digitalasset/daml/lf/engine/script/test/TlsIT.scala

```diff
@@ -10,47 +10,99 @@ import com.daml.ledger.api.testing.utils.SuiteResourceManagementAroundAll
 import com.daml.ledger.api.tls.TlsConfiguration
 import com.daml.lf.data.Ref._
 import com.daml.lf.engine.script.ScriptTimeMode
+import com.daml.lf.speedy.SValue
 import org.scalatest._
 
+import scala.concurrent.Future
+
 final class TlsIT
     extends AsyncWordSpec
     with SandboxParticipantFixture
     with Matchers
-    with SuiteResourceManagementAroundAll {
+    with SuiteResourceManagementAroundAll
+    with OCSPResponderFixture {
+
   val (dar, envIface) = readDar(stableDarFile)
 
-  val List(serverCrt, serverPem, caCrt, clientCrt, clientPem) = {
-    List("server.crt", "server.pem", "ca.crt", "client.crt", "client.pem").map { src =>
+  val List(
+    serverCrt,
+    serverPem,
+    caCrt,
+    clientCrt,
+    clientPem,
+    index,
+    ocspKey,
+    ocspCert,
+    clientRevokedCrt,
+    clientRevokedPem) = {
+    List(
+      "server.crt",
+      "server.pem",
+      "ca.crt",
+      "client.crt",
+      "client.pem",
+      "index.txt",
+      "ocsp.key.pem",
+      "ocsp.crt",
+      "client-revoked.crt",
+      "client-revoked.pem").map { src =>
       Some(new File(rlocation("ledger/test-common/test-certificates/" + src)))
     }
   }
 
+  val indexPath = index.get.getAbsolutePath
+  val caCertPath = caCrt.get.getAbsolutePath
+  val ocspKeyPath = ocspKey.get.getAbsolutePath
+  val ocspCertPath = ocspCert.get.getAbsolutePath
+  val clientCertPath = clientCrt.get.getAbsolutePath
+
   override def timeMode = ScriptTimeMode.WallClock
 
   override protected def config =
     super.config
-      .copy(tlsConfig = Some(TlsConfiguration(enabled = true, serverCrt, serverPem, caCrt)))
+      .copy(
+        tlsConfig = Some(
+          TlsConfiguration(enabled = true, serverCrt, serverPem, caCrt, revocationChecks = true)))
 
   "DAML Script against ledger with TLS" can {
     "test0" should {
```

Comment:
What is "test0"? Can we perhaps delete this and just make it:

```scala
   "DAML Script against ledger with TLS" should {
```

Reply:
This is the name of one of test scripts from `ScriptTest.daml`. I think we can change this as you suggest because the script name is not relevant here.